packages/schemas/src/types/log/interaction.ts

```diff
@@ -16,6 +16,7 @@ export enum Method {
   Password = 'Password',
   VerificationCode = 'VerificationCode',
   Social = 'Social',
+  SocialIdentity = 'SocialIdentity',
```

Comment:
Is this duplicated?

Reply:
maybe a better name could be more clear. 

This one represents the identifier of a social provided email or phone. Like, binding social with an existing email account flow, as we discussed, the identifier will indicate the user sign-in using a social verified trusted email address and then binds this social account. 
 
The identifier payload will be like `{connectorId: XXXX, indentityType: email}`. With a pre-verified social identifier and trusted email address in the interaction storage, this will be verified as an email identifier, just like verification code verified email.
 
This log method type represents this flow.

Reply:
could split to SocialEmail / SocialPhone, or any better name is welcomed.

Reply:
Removed